src/app/stock/stock.component.ts

```diff
@@ -1,88 +1,44 @@
+  styleUrls: ['./stock.component.css'],
 })
 export class StockComponent {
   displayPackage: boolean = false;
+  applyBlur: boolean = false;
   tableWrapperClass: string = 'table-wrapper';
   locationFilter: string = '';
+  packageList: Packaging[] = [];
+  locationList: Location[] = [];
+  stockList: Stock[] = [];
+  locationNames: string[] = [];
 
-  stock: Packaging[] = [
-    new Packaging('dozen', '1', 2, 1, 'Rode Doos1', 'Rotterdam'),
-    new Packaging('dozen', '2', 20, 10, 'Blauwe Doos2', 'Amsterdam'),
-    new Packaging('dozen', '1', 2, 1, 'Rode Doos3', 'Rotterdam'),
-    new Packaging('dozen', '2', 20, 10, 'Blauwe Doos4', 'Amsterdam'),
-    new Packaging('dozen', '1', 2, 1, 'Rode Doos5', 'Rotterdam'),
-    new Packaging('dozen', '2', 20, 10, 'Blauwe Doos6', 'Amsterdam'),
-    new Packaging('dozen', '1', 2, 1, 'Rode Doos7', 'Rotterdam'),
-    new Packaging('dozen', '2', 20, 10, 'Blauwe Doos8', 'Amsterdam'),
-    new Packaging('dozen', '1', 2, 1, 'Rode Doos9', 'Rotterdam'),
-    new Packaging('dozen', '2', 20, 10, 'Blauwe Doos10', 'Amsterdam'),
-    new Packaging('dozen', '1', 2, 1, 'Rode Doos11', 'Rotterdam'),
-    new Packaging('dozen', '2', 20, 10, 'Blauwe Doos12', 'Amsterdam'),
-    new Packaging('dozen', '1', 2, 1, 'Rode Doos13', 'Rotterdam'),
-    new Packaging('dozen', '2', 20, 10, 'Blauwe Doos14', 'Amsterdam'),
-    new Packaging('dozen', '1', 2, 1, 'Rode Doos15', 'Rotterdam'),
-    new Packaging('dozen', '2', 20, 10, 'Blauwe Doos16', 'Amsterdam'),
-    new Packaging('dozen', '1', 2, 1, 'Rode Doos17', 'Rotterdam'),
-    new Packaging('dozen', '2', 20, 10, 'Blauwe Doos18', 'Amsterdam'),
-    new Packaging('dozen', '1', 2, 1, 'Rode Doos19', 'Rotterdam'),
-    new Packaging('dozen', '2', 20, 10, 'Blauwe Doos20', 'Amsterdam'),
-    new Packaging('dozen', '1', 2, 1, 'Rode Doos21', 'Rotterdam'),
-    new Packaging('dozen', '2', 20, 10, 'Blauwe Doos22', 'Amsterdam'),
-    new Packaging('dozen', '1', 2, 1, 'Rode Doos23', 'Rotterdam'),
-    new Packaging('dozen', '2', 20, 10, 'Blauwe Doos24', 'Amsterdam'),
-    new Packaging('dozen', '1', 2, 1, 'Rode Doos25', 'Rotterdam'),
-    new Packaging('dozen', '2', 20, 10, 'Blauwe Doos26', 'Amsterdam'),
-    new Packaging('dozen', '1', 2, 1, 'Rode Doos27', 'Rotterdam'),
-    new Packaging('dozen', '2', 20, 10, 'Blauwe Doos28', 'Amsterdam'),
-    new Packaging('dozen', '1', 2, 1, 'Rode Doos29', 'Rotterdam'),
-    new Packaging('dozen', '2', 20, 10, 'Blauwe Doos30', 'Amsterdam'),
-    new Packaging('dozen', '1', 2, 1, 'Rode Doos31', 'Rotterdam'),
-    new Packaging('dozen', '2', 20, 10, 'Blauwe Doos32', 'Amsterdam'),
-    new Packaging('dozen', '1', 2, 1, 'Rode Doos33', 'Rotterdam'),
-    new Packaging('dozen', '2', 20, 10, 'Blauwe Doos34', 'Amsterdam'),
-  ];
+  private unsubscribe$ = new Subject<void>();
 
-  mockLocations: string[] = [
-    'Rotterdam', 'Amsterdam', 'Brabant'
-  ];
-
-  constructor(private http: DataStorageService){}
+  constructor(private http: DataStorageService) {}
```

Comment:
dit is niet de http service.
```suggestion
  constructor(private dataStorageService: DataStorageService) {}
```

Reply:
see commit b4bf2bc